apps/console/src/features/identity-verification-providers/components/forms/helpers/dynamic-ui-helper.tsx

```diff
@@ -272,20 +270,33 @@ export const performValidations = ( value: string, elementMetaData: InputFieldMe
 
     // perform required validation
     if (elementMetaData.required && !value) {
-        return ERROR_MSG_REQUIRED_FIELD;
+        return I18n.instance.t("console:develop.features.idvp.forms.dynamicUI.validations.required");
     }
 
     // skip the rest of the validations if the value is empty
     if (!value) {
         return;
     }
 
+    // perform range validation if the field is numeric
+    if (elementMetaData.type === FieldInputTypes.INPUT_NUMBER) {
+        const numericalValue: number = Number(value);
+
+        if (numericalValue < elementMetaData.minValue || numericalValue > elementMetaData.maxValue) {
```

Comment:
Just a reminder that with this logic if `numvericalValue` is equal to minValue or maxValue this if block will not be executed. If this intended ignore this comment.

Reply:
This is the intended behavior.